src/tests/class_tests/openms/source/TheoreticalSpectrumGenerator_test.cpp

```diff
@@ -359,19 +455,48 @@ END_SECTION
 
 START_SECTION(([EXTRA] bugfix test where losses lead to formulae with negative element frequencies))
 {
+  // this tests for the loss of CONH2 on Arginine, however it is not clear how
+  // this loss would occur in the first place.
   AASequence tmp_aa = AASequence::fromString("RDAGGPALKK");
```

Comment:
this does not make sense to me, it was introduced in 74e2ce6761e4a273164b29b8be487 and maybe @cbielow has some insight, but to me it seems like its an error in Residues.xml -- see #4448

Reply:
since the losses we look at here are amino-acid specific losses, these should all be part of the side chain and not the peptide back bone